doc/sphinx/proofs/writing-tactics/index.rst

```diff
@@ -0,0 +1,35 @@
+.. _writing-tactics:
+
+===============
+Writing tactics
```

Comment:
Perhaps "Creating new tactics"?  Typing "reflexivity." in CoqIDE could be taken as "writing".